frontend/src/app/auth/pages/signin/signin.page.html

```diff
@@ -16,7 +16,7 @@
           <ion-item>
             <ion-label position="stacked">
               <b>Password</b>
-              <a routerLink="/recover" float-right>Forgot password?</a>
+              <a routerLink="/auth/forgot-password" float-right>Forgot password?</a>
```

Comment:
path was implemented as forgot-password by @didemongu probably, so I connect the link to the path